webroot/src/models/Licensee/Licensee.model.ts

```diff
@@ -230,8 +230,14 @@ export class LicenseeSerializer {
         };
 
         // In get-all responses, server only returns a license state, not the actual license objects
-        if (json.licenseJurisdiction) {
+        if (json.licenseJurisdiction && json.licenses && json.licenses.length < 2) {
             licenseeData.licenseStates.push(new State({ abbrev: json.licenseJurisdiction }));
+        } else if (json.licenses && json.licenses.length > 1) {
+            json.licenses.forEach((serverLicense) => {
+                if (serverLicense.status === 'active') {
+                    licenseeData.licenseStates.push(new State({ abbrev: serverLicense.jurisdiction }));
+                }
+            });
```

Comment:
More to test my understanding, but couldn't we just run this code in all cases? I assume now that we're returning all the licenses, `licenseJurisdiction` is sort of duplicative?